frontend/packages/design-system/src/themes/Button.ts

```diff
@@ -3,14 +3,16 @@ export default {
     major: ({ colorScheme }: any) => {
       return {
         backgroundColor: `${colorScheme}.500`,
+
         _text: {
           color: 'white',
           bold: true,
         },
         _hover: {
-          backgroundColor: `${colorScheme}.100`,
+          backgroundColor:
+            colorScheme === 'secondary' ? 'link' : `${colorScheme}.200`,
```

Comment:
Not sure what you or the designers think about this, but I feel this should be `secondary.xxx` as the intention of this component isn't a "link" it's a button. I'm not sure the style of the button should be tied to the style of a link.